File: airesources/CSharp/Halite2/hlt/Entity.cs
namespace Halite2.hlt
{

    public class Entity : Position
    {

        private int owner;
        private int id;
        private int health;
        private double radius;

        public Entity(int owner, int id, double xPos, double yPos, int health, double radius)
            : base(xPos, yPos)
        {
            this.owner = owner;
            this.id = id;
            this.health = health;
            this.radius = radius;
        }

        public int GetOwner()
        {
            return owner;
        }

        public int GetId()
        {
            return id;
        }

        public int GetHealth()
        {
            return health;
        }

        public override double GetRadius()
        {
            return radius;
        }

        public override string ToString()
        {
            return "Entity[" +
                    base.ToString() +
                    ", owner=" + owner +
                    ", id=" + id +
                    ", health=" + health +
                    ", radius=" + radius +
                    "]";
        }
    }
}

File: airesources/Kotlin/src/kotlin/halite/Entity.kt
package halite

open class Entity(val owner: Int, val id: Int, xPos: Double, yPos: Double, val health: Int, val radius: Double) : Position(xPos, yPos) {

    override fun toString(): String {
        return "Entity[${super.toString()}, owner=$owner, id=$id, health=$health, radius=$radius]"
    }
}
